test/e2e/cypress/support/commands.js

```diff
@@ -92,6 +92,18 @@ Cypress.Commands.add('longRecoveryLifespan', ({} = {}) => {
     return config
   })
 })
+Cypress.Commands.add('dontLoginUserAfterRegistration', ({} = {}) => {
+  updateConfigFile((config) => {
+    delete config.selfservice.flows.registration.after['password']
+    return config
+  })
+})
+Cypress.Commands.add('enableLoginForVerifiedAddressOnly', ({} = {}) => {
+  updateConfigFile((config) => {
+    config.selfservice.flows.login['after'] = {password: {hooks: [{hook: 'require_verified_address'}]}}
+    return config
+  })
+})
```

Comment:
This looks good! However, I think this might need the negative as well? So iterating over `login.['after'].password.hooks` and `filter` out any hooks that are `require_verified_address`